exam1.py

```diff
@@ -0,0 +1,31 @@
+import sys
+sys.stdin = open("input1.txt","r")
+T = int(input())
+dic={}
+dic1 = {}
+record = []
+list1=[]
+for i in range(T):
+    a = input()
+    dic[a]=""
+    record = list(map(int,input().split()))
+    for j in record:
+        if j not in dic1:
+            dic1[j] = 1
+        else:
+            dic1[j] += 1
+
+    b = sorted(dic1.values(),reverse=True)
+    c = b[0]
+    for p in dic1:
+        if dic1[p] == c:
+            list1.append(p)
+            list1.sort(reverse=True)
+            p = list1[0]
+            dic[a] = p
```

Comment:
```suggestion
    for p in dic1:
        if dic1[p] == c:
            list1.append(p)
    p = max(list1)
    dic[a] = p
```
`for`문을 모두 돈 뒤 답을 구해도 됩니다!